pyiron/atomistics/structure/atoms.py

```diff
@@ -2598,6 +2598,14 @@ def get_distances_array(self, a0=None, a1=None, mic=True, vector=False):
         else:
             return dist
 
+    def append(self, atom):
+        if isinstance(atom, Atom) or isinstance(atom, ASEAtom):
+            super(Atoms, self).append(atom=atom)
+        else:
+            if atom.pbc.all() and np.isclose(atom.get_volume(), 0):
+                atom.cell = self.cell
```

Comment:
Doesn't this change the cell of the second structure? Maybe we should copy the second structure and then change the cell

Reply:
You mean `atom`? Changing the cell of that structure is what I want to achieve, so that's fine.

Reply:
> 
> 
> You mean `atom`? Changing the cell of that structure is what I want to achieve, so that's fine.

Yes but if I do `a.append(b)`, `b.cell` will now have the same value as `a.cell` when it should be 0. This shouldn't change during the operation right?

Reply:
Very good point!